$NetBSD: patch-Makefile,v 1.2 2021/01/16 12:20:00 nia Exp $

Honor CFLAGS/LDFLAGS.

Allow the default location of the X16 ROM to be overridden.

--- Makefile.orig	2019-12-20 20:00:49.000000000 +0000
+++ Makefile
@@ -10,8 +10,8 @@ else
 	SDL2CONFIG=sdl2-config
 endif
 
-CFLAGS=-std=c99 -O3 -Wall -Werror -g $(shell $(SDL2CONFIG) --cflags) -Iextern/include -Iextern/src
-LDFLAGS=$(shell $(SDL2CONFIG) --libs) -lm
+CFLAGS+=-std=c99 -Wall $(shell $(SDL2CONFIG) --cflags) -Iextern/include -Iextern/src
+LDFLAGS+=$(shell $(SDL2CONFIG) --libs) -lm
 
 OUTPUT=x16emu
 
@@ -48,6 +48,9 @@ ifneq ("$(wildcard ./rom_labels.h)","")
 HEADERS+=rom_labels.h
 endif
 
+ifdef ROM_RELATIVE_PATH
+	CFLAGS+=-DROM_RELATIVE_PATH=\"$(ROM_RELATIVE_PATH)\"
+endif
 
 all: $(OBJS) $(HEADERS)
 	$(CC) -o $(OUTPUT) $(OBJS) $(LDFLAGS)
